modules/qna/src/views/full/Components/ImportModal.tsx

```diff
@@ -0,0 +1,249 @@
+    const { data: status } = await props.axios.get(`/mod/qna/json-upload-status/${statusId}`)
+    setUploadStatus(status)
+
+    if (status === 'Completed') {
+      clearStatus()
+      closeDialog()
+      toastSuccess(lang.tr('module.qna.import.uploadSuccessful'))
+      props.onImportCompleted()
+    } else if (status.startsWith('Error')) {
+      clearStatus()
+      setHasError(true)
+    }
+  }
+
+  const readFile = (files: FileList | null) => {
+    if (files) {
+      setFile(files[0])
+      setFilePath(files[0].name)
+    }
+  }
+
+  const clearStatus = () => {
+    setStatusId(undefined)
+    setIsLoading(false)
+  }
+
+  const closeDialog = () => {
+    clearState()
+    props.toggle()
+  }
+
+  const clearState = () => {
+    setFilePath(undefined)
+    setFile(undefined)
+    setUploadStatus(undefined)
+    setStatusId(undefined)
+    setAnalysis(undefined)
+    setHasError(false)
+  }
+
+  const renderUpload = () => {
+    return (
+      <div
+        onDragOver={e => e.preventDefault()}
+        onDrop={e => {
+          e.preventDefault()
+          readFile(e.dataTransfer.files)
+        }}
+      >
+        <div className={Classes.DIALOG_BODY}>
+          <FormGroup
+            label={<span>{lang.tr('module.qna.import.selectJson')}</span>}
+            labelFor="input-archive"
+            helperText={<span>{lang.tr('module.qna.import.selectJsonHelp')}</span>}
+          >
+            <FileInput
+              text={filePath || lang.tr('chooseFile')}
+              onChange={e => readFile((e.target as HTMLInputElement).files)}
+              inputProps={{ accept: '.json' }}
+              fill
+            />
+          </FormGroup>
+        </div>
+        <div className={Classes.DIALOG_FOOTER}>
+          <div className={Classes.DIALOG_FOOTER_ACTIONS}>
+            <Button
+              id="btn-next"
+              text={isLoading ? lang.tr('pleaseWait') : lang.tr('next')}
+              disabled={!filePath || !file || isLoading}
+              onClick={analyzeImport}
+              intent={Intent.PRIMARY}
+            />
+          </div>
+        </div>
+      </div>
+    )
+  }
+
+  const renderAnalysis = () => {
+    const { qnaCount, cmsCount, fileQnaCount, fileCmsCount } = analysis
+
+    return (
+      <Fragment>
+        <div className={Classes.DIALOG_BODY}>
+          <div>
+            <p>
+              {lang.tr('module.qna.import.fileContains', {
+                fileQnaCount: <strong>{fileQnaCount}</strong>,
+                fileCmsCount: <strong>{fileCmsCount}</strong>
+              })}
+              <br />
+              <br />
+              {lang.tr('module.qna.import.botContains', {
+                qnaCount: <strong>{qnaCount}</strong>,
+                cmsCount: <strong>{cmsCount}</strong>
+              })}
+            </p>
+
+            <p style={{ marginTop: 30 }}>
+              <RadioGroup
+                label={lang.tr('module.qna.import.whatLikeDo')}
```

Comment:
I know you simply copy pasted, but just want to put some emphasis on that translation label > < whatLikeDo